d20 Modern/d20 Modern.html

```diff
@@ -382,7 +382,7 @@
                 </tr>
                 <tr>
                     <td><input type="checkbox" class="sheet-bigcheck" name="attr_Computer_Use-CrossClass" value="2"/><span></span></td>
-                    <td><button type="roll" class="sheet-btn" name="roll_Computer_Use" value="/e rolls Computer Use [[1d20+@{Computer_Use}]]" >Computer Use</button></td>
+                    <td><button type="roll" class="sheet-btn" name="roll_Computer_Use" value="/e rolls Computer Use [[1d20+@{Computer_Use}]]" ><div class="sheet-smaller">Computer Use</button></td>
```

Comment:
Could you please add closing div tags to these lines? Even if it technically works, that's still best practice.